etc/init

```diff
@@ -11,7 +11,7 @@ unset ml
 unset BASH_ENV
 
 # Provide a sane path within the container
-if [ -z ${SINGULARITYENV_PATH+x} ]; then
+if [ -z "${SINGULARITYENV_PATH:-}" ]; then
     SINGULARITYENV_PATH="/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
 else
     SINGULARITYENV_PATH="$SINGULARITYENV_PATH:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
```

Comment:
I think this whole `if` statement can be thrown away and replaced by a single line.  
```
SINGULARITYENV_PATH="{$SINGULARITYENV_PATH:-}:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
```
That should do the right thing, correct?

Reply:
With that formulation you'd have a lone : at start of it if the var isn't set when the test occurs.

Also, $ is outside of {

Reply:
Ah. Right you are. (On both counts of course).